python/llm/setup.py

```diff
@@ -44,7 +45,36 @@ def get_llm_packages():
     return llm_packages
 
 
+lib_urls = [
+    "https://sourceforge.net/projects/analytics-zoo/files/bigdl-llm/llama.dll",
+    "https://sourceforge.net/projects/analytics-zoo/files/bigdl-llm/quantize-llama.exe",
+    "https://sourceforge.net/projects/analytics-zoo/files/bigdl-llm/gptneox.dll",
+    "https://sourceforge.net/projects/analytics-zoo/files/bigdl-llm/quantize-gptneox.exe",
+    # TODO: add bloomz
+]
+
+
+def download_libs(url: str):
+    libs_dir = os.path.join(llm_home, "bigdl", "llm", "libs")
+    if not os.path.exists(libs_dir):
+        os.makedirs(libs_dir, exist_ok=True)
+    libso_file_name = url.split('/')[-1]
+    libso_file = os.path.join(libs_dir, libso_file_name)
+    if not os.path.exists(libso_file):
+        urllib.request.urlretrieve(url, libso_file)
+
+
 def setup_package():
+    
+    package_data = [
+        "libs/llama.dll",
+        "libs/quantize-llama.exe",
+        "libs/gptneox.dll",
```

Comment:
are we missing `quantize-gptneox.exe` here?

Reply:
yeah it's missing here